code/panels/UpdatePanel.php

```diff
@@ -83,28 +85,42 @@ public function getLatestSilverStripeVersion() {
 		return $latestVersion;
 	}
 
-	public function isCurrentSilverStripeVersion() {
-		if ($this->getCurrentSilverStripeVersion() && $this->getLatestSilverStripeVersion()) {
-			return $this->getCurrentSilverStripeVersion() >= $this->getLatestSilverStripeVersion();
+	public function isNewerSilverStripeVersion($versionNumber, $latestVersionNumber) {
+		if ($versionNumber == $latestVersionNumber) {
```

Comment:
Not so sure on this part. I get why you have it but it conflicts with the name of the function. Maybe the function needs to be reversed so it is `isOlderSilverStripeVersion`?